website/docs/reference/commands/rpc.md

```diff
@@ -210,8 +210,7 @@ All RPC requests accept the following parameters in addition to the parameters l
 
 Several of the following request types accept these additional parameters:
 - `threads`: The number of [threads](configure-your-profile#understanding-threads) to use when compiling (optional)
-- `models`: The space-delimited set of models to compile, run, or test (optional)
-- `select`: The space-delimeted set of resources to seed or snapshot (optional)
+- `select`: The space-delimeted set of resources to execute (optional). (`models` is also supported on some request types for backwards compatibility.)
```

Comment:
```suggestion
- `select`: The space-delimited set of resources to execute (optional). (`models` is also supported on some request types for backwards compatibility.)
```

Reply:
fixed in https://github.com/dbt-labs/docs.getdbt.com/pull/831/commits/fb40346c5952eb505efeca154d5cd6cb2e108a9d